### PR TITLE
Initial version 2.0.0, implemented the bidder with the new template

### PR DESCRIPTION
**Pull request by RajeshRhythmone**

### CODE REVIEW

**Comment by ix-certification** [comment]
We would like to provide 2 feedbacks.

1. In rhythm-one-htb-validator.js, it shows that "imp" should be a string. But in rhythm-one-htb-system-tests.js, the example config has "imp: ['1']", which doesn't match the validator.
2. In rhythm-one-htb-system-tests.js function getBidRequestRegex, the urlRegex doesn't match the actual bid request url on https site, which is causing 5 cases failing on https site. We recommend removing the http part in the regex, just replace it with a wild card.

Let us know if you have any question regarding the above feedbacks.

**Comment by RajeshRhythmone** [comment]
Updated your review comments, please check

File: rhythm-one-htb.js [hunk 1]
```diff
@@ -129,11 +130,13 @@ function RhythmOneHtb(configs) {
...
         /* ---------------------- PUT CODE HERE ------------------------------------ */
-        var queryObj = {};
+        var queryObj = [];
         var callbackId = System.generateUniqueId();
 
         /* Change this to your bidder endpoint. */
-        var baseUrl = Browser.getProtocol() + '//someAdapterEndpoint.com/bid';
+
+        var baseUrl;
+        var w = typeof window !== 'undefined' ? window : { document: { location: { href: '' } } };
```
Comment (ix-certification) [inline rhythm-one-htb.js]: We recommend using `Browser.topWindow` instead of `window`

**Comment by RajeshRhythmone** [comment]
Updated review comments, please check

**Comment by ix-certification** [comment]
We would like to provide 2 feedbacks.

1. Please update the adapter version in rhythm-one-htb.js. https://github.com/indexexchange/rhythm-one-adapter/blob/b9975752773ca2aeb5ecae4302232e4700a80e6d/rhythm-one-htb.js#L560 It needs to match the version in CHANGES.md
2. We added a bidder documentation in master-v2 branch, please fill out DOCUMENTATION.md to reflect adapter information.

Please let us know if you have any question regarding the above feedbacks.

**Comment by RajeshRhythmone** [comment]
Updated review comments and added DOCUMENTATION.md file, please check

**Comment by ix-certification** [comment]
Please resolve the conflicts for `DOCUMENTATION.md`

**Comment by ix-certification** [comment]
In `DOCUMENTATION.md`, please explicitly list `N` beside the fields under “General Compatibility” and “Browser Compatibility” which are not supported (left blank).

**Comment by ix-certification** [comment]
In DOCUMENTATION.md, please explicitly list “Won't Test” beside the fields that you are not sure if you support (left blank) under “Browser Compatibility”.

**Pull request merged**